src/sink.h

```diff
@@ -175,6 +175,11 @@ class Sink : public cyclus::Facility  {
                              "accept at each time step"}
   double capacity;
 
+  #pragma cyclus var {"default": "", "tooltip": "requested composition", \
+                      "doc": "name of recipe to use for material " \
+                             "requests"}
```

Comment:
Also some documentation that "an empty string (default) means to accept everything." That isn't intuitive from the interface.